lib/FFI/CheckLib.pm

```diff
@@ -57,6 +57,38 @@ our $system_path = [];
 our $os ||= $^O;
 my $try_ld_on_text = 0;
 
+sub _homebrew_lib_path {
+  require File::Which;
+  return undef unless File::Which::which('brew');
+  chomp(my $brew_path = (qx`brew --prefix`)[0]);
+  return "$brew_path/lib";
+}
+
+sub _macports_lib_path {
+  require File::Which;
+  my $port_path = File::Which::which('port');
+  return undef unless $port_path;
+  $port_path =~ s|bin/port|lib|;
+  return $port_path;
+}
+
+sub _darwin_extra_paths {
+  my $pkg_managers = lc( $ENV{FFI_CHECKLIB_PACKAGE} // 'homebrew,macports' );
```

Comment:
```suggestion
  my $pkg_managers = lc( $ENV{FFI_CHECKLIB_PACKAGE} || 'homebrew,macports' );
```

```
% perlver lib --blame      

 ------------------------------------------------------------
 File    : lib/FFI/CheckLib.pm
 Line    : 76
 Char    : 53
 Rule    : _perl_5010_operators
 Version : 5.010
 ------------------------------------------------------------
 //
 ------------------------------------------------------------
```

Reply:
Done!

Could you explain to me why `||` is preferred over `//`?

Reply:
I like `//` a lot, and would much prefer it!  But yeah, FFI::Platypus is key infrastructure supports 5.8.4 so this module has to also support 5.8.4 as well.

I understand the cases where `||` is different if we set it to a false value then we get the default.  In this case I think that is fine.